_posts/2017/2017-03-30-finding-the-best-vindinium-bot-written-in-elixir.md

```diff
@@ -0,0 +1,51 @@
+---
+layout: post
+title: "Finding the best vindinium bot written in Elixir"
+date: "2017-03-30 06:39:20"
+image: blog-header/elixir.jpg
+categories: events
+authors: ["Rose"]
+---
+<style>
+.twitter-tweet {
+  margin: auto;
+}
+</style>
+
+This Tuesday, the second part of the [hh.ex vindinium challenge](https://www.meetup.com/de-DE/Elixir-Hamburg/events/238421135/?eventId=238421135) took place in the [ePages](https://www.epages.com/en/) office in Hamburg.
+Despite the amazing weather conditions outside, there were quite a few [Elixir](http://elixir-lang.org/) and [vindinium](http://vindinium.org/) interested developers to join the meetup.
+
+
+## Programming a semi static blogengine with Elixir
+
+The event started off with a presentation about our developer Erik's side project to create a semi static blogengine called Jelly Shot.
```

Comment:
The event started off with a presentation about a side project of our developer Erik. He presented his semi static blog engine "Jelly Shot".